packages/size-limit/size-limit-error.js

```diff
@@ -37,7 +37,7 @@ const MESSAGES = {
   unknownArg: arg =>
     `Unknown argument *${arg}*. Check command for typo and read docs.`,
   unknownEntry: entry =>
-    `Size Limit didn’t find *${entry}* entry in custom Webpack config`,
+    `Size Limit didn’t find *${entry}* entry in the custom bundler config`,
```

Comment:
Spent some time in confusion how esbuild change may break anything with webpack config. Let's rename the error message to a more generic